src/index/PermutationExporterMain.cpp

```diff
@@ -4,14 +4,15 @@
 #include "./Index.h"
 #include "./IndexImpl.h"
 #include "./TriplesView.h"
+#include "util/MemorySize/MemorySize.h"
 
 /// Dump a  certain permutation to stdout in a human-readable way as IDs, and
 /// in deterministic order
 template <typename Permutation>
 void dumpToStdout(const Permutation& permutation) {
   ad_utility::AllocatorWithLimit<Id> allocator{
       ad_utility::makeAllocationMemoryLeftThreadsafeObject(
-          std::numeric_limits<uint64_t>::max())};
+          ad_utility::MemorySize::bytes(std::numeric_limits<uint64_t>::max()))};
```

Comment:
```suggestion
          ad_utility::MemorySize:max())};
```

Reply:
Ah, small problem with that: `uint64_t` and `size_t` don't always have the same size.

Reply:
Done, see [3977bd5](https://github.com/ad-freiburg/qlever/pull/1112/commits/3977bd5ab12a15cd0a22bf26be9fa7473b1427e1).